site/src/pages/ManagementSettingsPage/IdpSyncPage/IdpSyncPageView.tsx

```diff
@@ -135,27 +216,50 @@ const IdpField: FC<IdpFieldProps> = ({
 }) => {
 	return (
 		<span css={{ display: "flex", alignItems: "center", gap: "16px" }}>
-			<h4>{name}</h4>
-			<p css={styles.field}>
-				{fieldText ||
-					(showStatusIndicator && (
-						<div
-							css={{
-								display: "flex",
-								alignItems: "center",
-								gap: "8px",
-								height: 0,
-							}}
-						>
-							<StatusIndicator color="error" />
-							<p>disabled</p>
-						</div>
-					))}
-			</p>
+			<p css={styles.fieldLabel}>{name}</p>
+			{fieldText ? (
+				<p css={styles.fieldText}>{fieldText}</p>
+			) : (
+				showStatusIndicator && (
```

Comment:
Is this the right name for the prop if it's only being used to show disabled statuses?

Reply:
I though it may be used for more in the future but Ill rename it for now.